docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -31,4 +31,8 @@ in ClusterAPI are kept in sync with the versions used by `sigs.k8s.io/controller
 
 ### Other
 
--
+- Logging:
+    - To align with the upstream Kubernetes community CAPI now configures logging via `component-base/logs`. 
+      This provides advantages like support for the JSON logging format (via `--logging-format=json`) and automatic
+      deprecation of klog flags aligned to the upstream Kubernetes deprecation period. For more details see [6072](https://github.com/kubernetes-sigs/cluster-api/pull/6072)
```

Comment:
Note: Pending https://github.com/kubernetes-sigs/cluster-api/pull/6190 we should link https://github.com/kubernetes-sigs/cluster-api/pull/6190 here too